indico/modules/rb_new/client/js/common/rooms/selectors.js

```diff
@@ -15,15 +15,84 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
+import _ from 'lodash';
 import {createSelector} from 'reselect';
 
 import {RequestState} from 'indico/utils/redux';
 
 
 export const hasLoadedEquipmentTypes = ({rooms}) => rooms.requests.equipmentTypes.state === RequestState.SUCCESS;
-export const getEquipmentTypes = ({rooms}) => rooms.equipmentTypes;
+const getEquipmentTypes = ({rooms}) => rooms.equipmentTypes;
+const getFeaturesMapping = createSelector(
+    getEquipmentTypes,
+    equipmentTypes => {
+        const features = {};
+        equipmentTypes
+            .forEach(({name: eqName, features: eqFeatures}) => {
```

Comment:
Honestly, it is not more readable to me. Why did you put it in the next line? It seems like it could fit perfectly fine in the previous one - that would save one indentation level also.